### DISCUSSION TITLE
SurrealDB adapter

### DISCUSSION THREAD

**Discussion by jquesada2016** [nextauthjs/next-auth#7526] [Ideas]
### Description 📓

Provide an adapter for SurrealDB.

### How to reproduce ☕️

N/A

### Contributing 🙌🏽

Yes, I am willing to help implement this feature in a PR

**Comment by hgoona** [comment]
a SurrealDB Adapter for authjs/SvelteKit would be great! I dunno how to put this together at present...

**Comment by auipga** [comment]
`npm i @auth/surrealdb-adapter`
https://authjs.dev/reference/adapter/surrealdb
https://github.com/nextauthjs/next-auth/tree/main/packages/adapter-surrealdb

It's kinda outdated. It requires to downgrade surrealdb.js but it works
```diff
   "dependencies": {
+    "@auth/surrealdb-adapter": "^0.2.5",
+    "surrealdb.js": "^0.9.1",
-    "surrealdb.js": "^0.11.0",
```

**Reply by ManAnRuck** [reply]
PR working with new version is open and testable :)

https://github.com/nextauthjs/next-auth/pull/9700
